package.json

```diff
@@ -15,23 +13,37 @@
     "jquery": "^2.1.1"
   },
   "devDependencies": {
+    "angular-mocks": "^1.4.0",
+    "babel-loader": "^5.1.2",
     "chai": "^1.9.1",
+    "css-loader": "^0.18.0",
     "eslint": "^1.1.0",
     "eslint-loader": "^1.0.0",
     "eslint-plugin-angular": "^0.4.0",
+    "file-loader": "^0.8.4",
     "html-loader": "^0.3.0",
     "json-loader": "^0.5.2",
-    "karma": "^0.12.16",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^0.1.4",
-    "karma-firefox-launcher": "^0.1.3",
-    "karma-mocha": "^0.1.3",
-    "karma-sinon": "^1.0.3",
+    "jasmine-core": "^2.3.4",
+    "karma": "^0.13.8",
+    "karma-chrome-launcher": "^0.1.12",
+    "karma-firefox-launcher": "^0.1.6",
+    "karma-html-reporter": "^0.2.6",
+    "karma-jasmine": "^0.3.6",
+    "karma-junit-reporter": "^0.2.2",
+    "karma-phantomjs-launcher": "^0.2.0",
+    "karma-sourcemap-loader": "^0.3.5",
+    "karma-verbose-reporter": "0.0.2",
+    "karma-webpack": "^1.7.0",
+    "phantomjs-polyfill": "0.0.1",
+    "requirejs": "^2.1.18",
+    "less": "^2.5.1",
+    "less-loader": "^2.2.1",
     "mocha": "^1.18.2",
     "protractor": "^0.22.0",
     "sinon": "^1.9.1",
     "sinon-chai": "^2.5.0",
-    "webpack": "^1.11.0",
-    "webpack-dev-server": "^1.10.1"
```

Comment:
Should both be devDependencies. The scripts block should then reference webpack from the nodes_modules dir. This why you don't have to install all the tools on your build server, but can rather use it from the project dir.